blueocean-dashboard/src/main/js/LoadingIndicator.js

```diff
@@ -44,7 +44,10 @@ export default {
     },
 
     hide() {
-        loadingCount--;
```

Comment:
I think it is reasonable that people may go directly right? if it has a URI, it can be bookmarked and shared.